docs/realm.js

```diff
@@ -274,7 +274,13 @@ class Realm {
     removeAllListeners(name) { }
 
     /**
-     * Synchronously call the provided `callback` inside a write transaction.
+     * Synchronously call the provided `callback` inside a write transaction. If an exception happens inside a transaction,
+     * you’ll lose the changes in that transaction, but the Realm itself won’t be affected (or corrupted). The exception is
+     * rethrown. More precisely, {@link Realm#beginTransaction beginTransaction()} and {@link Realm#commitTransaction commitTransaction()} will be called
+     * automatically. If any exception is thrown during the transaction {@link Realm#cancelTransaction cancelTransaction()} will
+     * be called instead of {@link Realm#commitTransaction commitTransaction()}.
```

Comment:
```suggestion
     * be called instead of {@link Realm#commitTransaction commitTransaction()} and the exception will be re-thrown to the caller of `write`.
```